### ISSUE TITLE
Any reason why I can't go to the Floating Islands from the Den, Kitchen or Garden?

### ISSUE THREAD

**Issue by Justformas** [tw4449-s-MAS-Submods/tw4449-Custom-Room-Selection-Pack-Main-Repository#39]
I can ask Monika to go to the Floating Islands from any of the Spacerooms including the original one, but the option doesn't show up when I'm in the Den, Kitchen, or Garden. It's not a huge deal, but is this a bug, limitation, or is there some other reason?

**Comment by Cdino11** [comment]
Yes there is a reason. The other rooms aren't spaceroom variants like v1-3. V1-3 are the spaceroom with different furniture setups, while the others are different rooms entirely. Think of it like different areas of a house (you can't see the everything outside from every room). So the floating islands are only visible from the spaceroom

**Comment by Justformas** [comment]
Oh I guess I viewed the floating islands as another location to actually "go" to, not just look at. Though you can't actually move around the islands, that's the case for the spaceroom too. The difference is just that you get to change your view when "looking" at the islands. But there I was immersing myself more than the mod itself intended I guess.

But anyway in that case, in theory, there isn't really anything stopping the mod having you and Monika automatically taking that extra step is there? I personally would consider it an implied step being taken anyway. By that I mean, when you click the option to see the floating islands from some other room, you would be implied to be "going back to the spaceroom" to check the islands, and then when you're done, you would be implied to be returning to the room you were last at.
